plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -103,18 +103,17 @@ private static String prepareCassandraYaml()
     {
         String original = Resources.toString(getResource("cu-cassandra.yaml"), UTF_8);
 
-        File tempDirFile = createTempDir();
-        tempDirFile.deleteOnExit();
-        Path tmpDirPath = tempDirFile.toPath();
+        Path tmpDirPath = createTempDirectory(null);
```

Comment:
Pls add logic for deleting the temporary directory in the `#close()` method.

You can use `io.trino.plugin.elasticsearch.ElasticsearchServer` as template for this change.

Reply:
This should be covered by `yamlFile.deleteOnExit();`